test/js/node/http/node-fetch.test.js

```
@@ -91,6 +91,41 @@ test("node-fetch uses node streams instead of web streams", async () => {
+  });
+  const webBody = Object.getOwnPropertyDescriptor(originalResponse.prototype, "body").get;
+
+  for (const [path, init, status] of [
+    ["/204", {}, 204],
+    ["/", { method: "HEAD" }, 200],
+  ]) {
+    const result = await fetch2(new URL(path, server.url), init);
+    expect(result.status).toBe(status);
+    expect(webBody.call(result)).toBeNull();
+    expect(result.body).toBeInstanceOf(stream.Readable);
+    expect(result.body === result.body).toBe(true); // cached lazy getter
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)